packages/import/src/importers/jiraCsv/JiraCsvImporter.ts

```diff
@@ -78,6 +80,9 @@ export class JiraCsvImporter implements Importer {
           : url
           ? `[View original issue in Jira](${url})`
           : undefined;
+      const estimate = row["Custom field (Story point estimate)"]
```

Comment:
Even though you declared the type above, since it's coming from a string, there's no validation that these numbers are actually one of the Fibonacci estimates, so we have to make sure we round the  estimate to the nearest Fibonacci value, not round the number in general